### DISCUSSION TITLE
Foutmelding tijden installatie

### DISCUSSION THREAD

**Discussion by rvos1975** [fabianishere/udm-iptv#220] [Q&A]
### What is your ISP? (if relevant)

KPN

### Diagnostic Information

![udm-iptv-diag](https://user-images.githubusercontent.com/123396116/214093681-5139b8e7-8513-4b32-9909-abfdcc97d04d.PNG)

![Foutmelding](https://user-images.githubusercontent.com/123396116/214092502-0f35a53e-1a0c-4d44-b385-32234047a4b0.PNG)

### Describe the Bug

Ik heb geen beeld op de tv krijg onderstaande melding bij het installeren.
kan iemand mij helpen?

### Expected Behavior

dacht als dit ingesteld was dat ik TV kon kijken

**Comment by fabianishere** [comment]
Please try

```
rm /etc/udm-iptv.conf
touch /etc/udm-iptv.conf

dpkg-reconfigure -p medium udm-iptv
```

**Comment by rvos1975** [comment]
root@ubnt:/# rm /etc/udm-iptv.conf
root@ubnt:/# touch /etc/udm-iptv.conf
root@ubnt:/#
root@ubnt:/# dpkg-reconfigure -p medium udm-iptv
/usr/sbin/dpkg-reconfigure: udm-iptv is broken or not fully installed
root@ubnt:/#

**Reply by fabianishere** [reply]
Please run the installation script again.

**Comment by rvos1975** [comment]
Gedaan en de TV werkt nu wel.

Dank je wel
